src/main/kotlin/io/github/moulberry/notenoughupdates/miscgui/DynamicLightItemsEditor.kt

```diff
@@ -77,6 +83,40 @@ class DynamicLightItemsEditor() : GuiScreen() {
 
         fontRendererObj.drawString("Dynamic Light Items Editor", guiLeft + 10, guiTop + 7, 4210752)
 
+        fontRendererObj.drawString("Add Held Item", guiLeft + 27, guiTop + ySize - 26, 4210752)
+        fontRendererObj.drawString("Remove Item", guiLeft + 130, guiTop + ySize - 26, 4210752)
+
+        GlStateManager.color(1f, 1f, 1f, 1f)
+        Minecraft.getMinecraft().textureManager.bindTexture(help)
+        Utils.drawTexturedRect((guiLeft + xSize + 3).toFloat(), guiTop.toFloat(), 16F, 16F, GL11.GL_NEAREST)
+        if (mouseX >= guiLeft + xSize + 3 &&
+            mouseX <= guiLeft + xSize + 19 &&
+            mouseY >= guiTop &&
+            mouseY <= guiTop + 16) {
+            val tooltip = listOf(
+                "§bDynamic Light Item Editor",
+                "§eWhat is this?",
+                "§eNEU makes use of OptiFine's feature of certain items",
+                "§eemitting dynamic light. By default OptiFine only implements",
+                "§ethis feature for a select few minecraft items.",
+                "",
+                "§eThis editor however, allows you to add specific skyblock",
+                "§eitems that will emit dynamic light when held. Simply hold the",
+                "§eitem you wish to add, then open this menu again and click",
+                "§e'Add Held Item', now if you have OptiFine installed and the",
+                "§edynamic lights option enabled, the added items will emit light!",
+                "",
+                "§eTo remove an item, click the item in this menu and click",
+                "§ethe 'Remove Item' button in the bottom right.",
+            )
+            Utils.drawHoveringText(tooltip, mouseX, mouseY, width, height, -1)
+        }
+
+        if (!didApplyMixin) {
+            fontRendererObj.drawString("Could not apply OptiFine mixin!", guiLeft + 30, guiTop + 22, Color.RED.rgb)
```

Comment:
Average person doesnt know what "mixin" means, probably say something like Couldn't find optifine ask for support in the discord

Reply:
Good point, I'll change that rq